modules/ob-creating-a-build-with-oci-artifacts.adoc

```diff
@@ -0,0 +1,96 @@
+// This module is included in the following assembly:
+//
+// * work-with-builds/using-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="Creating-a-build-with-OCI-artifacts_{context}"]
+= Creating a build with OCI artifacts
+
+You can create a build using an Open Container Initiative(OCI) artifact as your source code. It is also known as a scratch image. It is an empty base image used to create small and secure container images by adding only the application and its essential dependencies. You can pull the OCI artifact from a container registry and then push the resulting image to a target registry.
+
+
+.Prerequisites
+
+* You have installed the {builds-operator} on the {ocp-product-title} cluster.
+* You have installed the `oc` CLI.
+* Optional: You have installed the link:https://console.redhat.com/openshift/downloads[`shp` CLI].
+
+.Procedure
+
+. Create a `Build` resource and apply it to the {ocp-product-title} cluster by using the following example configuration:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
+apiVersion: shipwright.io/v1beta1
+kind: Build
+metadata:
+  name: my-oci-build <1>
+spec:
+  source:
+    type: OCI
+    ociArtifact:
+      image: <quay.io/org/image:tag> <2>
+  strategy:
+    name: <strategy-name> <3>
+    kind: ClusterBuildStrategy
+  paramValues: 
+    - name: <parameter-name> <4> 
+      value: <parameter-value> <5>
+  output: 
+    image: <target-image-registry/repository/image:tag> <6>
+    pushSecret: <secret-name-for-credentials> <7>
+
+EOF
+----
+<1> Name of the `Build` resource.
+<2> Location where the OCI Artifact source image reference is placed.
+<3> Name of the build strategy that you want to use to build the container. (`buildah` or `source-to-image`) 
+<4> Name of parameter to be configure in the build strategy. 
+<5> Location of the builder image required to build the output image. 
+<6> Location where you want to push the built image.
+<7> Optional: Secret name that stores the credentials for pushing container images. To generate a secret for a private registry for authentication, see "Authentication to container registries".
+
+. Upload the source code to an OCI artifact by using one of the methods:
+
+.*Using the shp CLI:*
+
+. Run the following command to package the local source code into a scratch container image, push to the designated registry and run the build:
++
+[source,terminal]
+----
+$ shp build upload my-oci-build <1>
+----
+<1> Name of the `Build` resource.
+
+. Run the build and upload the package using the following command:
++
+[source,terminal]
+----
+$ shp build upload
+----
```

Comment:
I think only the first step is enough. I have to check to confirm. Will get back on this.